### PR TITLE
add eja micro web server

### PR DESCRIPTION
**Pull request by ubaldus**
Adding package **eja** from debian _web_ subsection.

### CODE REVIEW

File: packages/eja/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Skeleton build.sh script for new package.
+# For reference about available fields, check the Termux Developer's Wiki page:
+# https://github.com/termux/termux-packages/wiki/Creating-new-package
+
```
Comment (Grimler91) [inline packages/eja/build.sh:4]: Please delete the header

File: packages/eja/build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+TERMUX_PKG_HOMEPAGE=http://eja.it
```
Comment (Grimler91) [inline packages/eja/build.sh:5]: use https instead of http, since https://eja.it seem to exist

File: packages/eja/build.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+TERMUX_PKG_DESCRIPTION="eja micro web server"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=13.11.20
+TERMUX_PKG_SRCURL=https://github.com/eja/eja/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=d19a8071d524373bcfceb5dec834546e2c66108593f0bf54ef984d8a0819e050
+TERMUX_PKG_EXTRA_MAKE_ARGS="DESTDIR=$TERMUX_PREFIX prefix=/"
```
Comment (Grimler91) [inline packages/eja/build.sh:11]: Why setting `DESTDIR=$TERMUX_PREFIX` and `prefix=/`? I think build should work if you remove this line

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Hi, see comments

**Comment by ubaldus** [comment]
ok, thank you for checking.

**Comment by ubaldus** [comment]
the extra make argument is needed to allow correct installation of the manual.

**Comment by Grimler91** [comment]
> the extra make argument is needed to allow correct installation of the manual.

Alright, so it is because prefix, [as given as environmental var](https://github.com/termux/termux-packages/blob/273a6ff7a1aa68c9bdd5f30693ec40940b5b559c/scripts/build/termux_step_setup_variables.sh#L89), [is overwritten in the Makefile](https://github.com/eja/eja/blob/52408581889cd2ec4414c1bd7da0212a51752fcc/Makefile#L1). 

You don't need to set DESTDIR, `TERMUX_PKG_EXTRA_MAKE_ARGS="prefix=$TERMUX_PREFIX"` is enough. (Another possibility would be update the Makefile to have `prefix ?= /usr/local`, then `TERMUX_PKG_EXTRA_CONFIGURE_ARGS` would not have to be set).

**Comment by xeffyr** [comment]
As it seems everything installed to correct place, I'll merging this.
```
drwx------ builder/builder   0 2020-11-20 18:31 ./data/data/com.termux/files/usr/
drwx------ builder/builder   0 2020-11-20 18:31 ./data/data/com.termux/files/usr/bin/
-rwx------ builder/builder 414472 2020-11-20 18:31 ./data/data/com.termux/files/usr/bin/eja
drwx------ builder/builder      0 2020-11-20 18:31 ./data/data/com.termux/files/usr/share/
drwx------ builder/builder      0 2020-11-20 18:31 ./data/data/com.termux/files/usr/share/doc/
drwx------ builder/builder      0 2020-11-20 18:31 ./data/data/com.termux/files/usr/share/doc/eja/
lrwxrwxrwx builder/builder      0 2020-11-20 18:31 ./data/data/com.termux/files/usr/share/doc/eja/LICENSE -> ../../LICENSES/GPL-3.0.txt
drwx------ builder/builder      0 2020-11-20 18:31 ./data/data/com.termux/files/usr/share/man/
drwx------ builder/builder      0 2020-11-20 18:31 ./data/data/com.termux/files/usr/share/man/man1/
-rwx------ builder/builder    858 2020-11-20 18:31 ./data/data/com.termux/files/usr/share/man/man1/eja.1.gz
```

All further improvements can be done later, if necessary.

**Pull request merged**